src/pages/home.html

```diff
@@ -0,0 +1,1756 @@
+            alt="گالری عکس"
+            class="w-[328px] object-contain object-center"
+          />
+        </div>
+        <div
+          class="relative z-10 flex items-center justify-center bg-link-water"
+        >
+          <div class="relative">
+            <img
+              src="/src/assets/images/home/blade.png"
+              alt="عکس تیغه"
+              class="absolute -left-[13%] top-0 -z-10"
+            />
+            <img
+              src="/src/assets/images/flower.png"
+              alt="عکس گل"
+              class="w-[490px] object-cover object-center"
+            />
+          </div>
+          <div class="flex flex-col justify-start gap-6 pl-[9rem] pr-10">
+            <h3
+              class="flex flex-col whitespace-nowrap text-[39px] font-semibold leading-[55px] text-black"
+            >
+              <span>عکس ماندگار،</span>
+              <span>هدیه‌ای ارزشمند</span>
+            </h3>
+            <p
+              class="flex flex-col whitespace-nowrap text-base leading-8 text-bali-hai"
+            >
+              <span>با چاپ عکس‌های خود</span>
+              <span>همیشه خاطرات روزهای خوب و زیبایتان را</span>
+              <span>زنده و ماندگار نگه دارید</span>
+            </p>
+
+            <a
+              href="#?"
+              class="z-10 flex items-center justify-center gap-3 bg-bondi-blue bg-line-pattern px-5 py-5.5 text-link-water"
+            >
+              <p class="text-lg font-bold">چاپ آنلاین عکس</p>
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <circle
+                  cx="12"
+                  cy="12"
+                  r="9"
+                  stroke="#ffffff"
+                  stroke-width="1.5"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                />
+                <path
+                  d="M12 9L9 12M9 12L12 15M9 12H15"
+                  stroke="#ffffff"
+                  stroke-width="1.5"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                />
+              </svg>
+            </a>
+          </div>
+          <img
+            src="/src/assets/images/home/asterisk.png"
+            alt="عکس *"
+            class="absolute bottom-0 left-0 w-[91px]"
+          />
+        </div>
+      </section>
+      <!-- gallery section -->
+
+      <!-- services section -->
+      <section class="mx-auto mb-20 max-w-7xl">
+        <div class="flex items-center justify-center">
+          <div class="relative py-8">
+            <img
+              src="/src/assets/images/home/oranges.png"
+              alt=""
+              class="absolute right-0 top-0 w-[35px] translate-x-full"
+            />
+            <h3 class="text-5xl font-bold">خدمات ما</h3>
+          </div>
```

Comment:
I realized we can achieve this output without absolute positioning!

```suggestion
          <div class="py-8 flex flex-col justify-start">
            <img
              src="/src/assets/images/home/oranges.png"
              alt=""
              class="w-[35px] translate-x-full"
            />
            <h3 class="text-5xl font-bold">خدمات ما</h3>
          </div>
```